components/modal/confirm.tsx

```diff
@@ -76,18 +71,34 @@ export default function confirm(config) {
     [`${prefixCls}-${props.type}`]: true,
   }, props.className);
 
+  const dialogProps = assign({
```

Comment:
This code style is weird...

Just follow the old style. Just pass `zIndex` to `rc-dialog` is enough..